src/main/scala/ztis/cassandra/CassandraClient.scala

```diff
@@ -0,0 +1,122 @@
+package ztis.cassandra
+
+import java.net.InetSocketAddress
+import java.util
+
+import com.datastax.driver.core.policies.RoundRobinPolicy
+import com.datastax.driver.core.{Cluster, ResultSet}
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.slf4j.StrictLogging
+import ztis.UserAndRating
+
+import scala.collection.JavaConverters._
+
+class CassandraClient(config: Config) extends StrictLogging {
+
+  private[cassandra] val keyspace: String = CassandraClient.keyspace(config)
```

Comment:
I would extract configuration case class. We are mixing configuration parsing and simple CQL actions here.